x-pack/plugins/security_solution/server/search_strategy/security_solution/factory/network/top_n_flow/query.top_n_flow_network.dsl.ts

```diff
@@ -79,11 +69,62 @@ export const buildTopNFlowQuery = ({
   return dslQuery;
 };
 
+export const buildTopNFlowCountQuery = ({
```

Comment:
`buildTopNFlowQuery` and `buildTopNFlowCountQuery` is quite similar, the main difference between the two would be body.aggregations. Shall we define a function or const for the identical part and reuse them?